tests/DoctrineORMModuleTest/Yuml/YumlControllerFactoryTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace DoctrineORMModuleTest\Yuml;
+
+use DoctrineORMModule\Yuml\YumlController;
+use DoctrineORMModule\Yuml\YumlControllerFactory;
+use Zend\ServiceManager\AbstractPluginManager;
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+class YumlControllerFactoryTest extends \PHPUnit_Framework_TestCase
+{
+    public function testCreateService()
+    {
+        $config = [
+            'zenddevelopertools' => [
+                'toolbar' => [
+                    'enabled' => true
+                ]
+            ]
+        ];
+
+        $serviceLocator = $this->getMockBuilder(ServiceLocatorInterface::class)
+            ->getMock();
+        $pluginManager = $this->getMockBuilder(AbstractPluginManager::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $serviceLocator->expects(static::any())->method('get')->with('config')->willReturn($config);
```

Comment:
Not sure if these tests should have `expects` at all. This is not part of the scope of the test, so I think it should be removed (this is why I was suggesting `prophesies`) - it doesn't matter how many times `->get('config')` / `-> getServiceLocator` / ...  will be called etc